styles.css

```diff
@@ -138,4 +138,21 @@ header img {
 	display: flex;
 	flex-flow: row wrap;
 	justify-content: space-between;
+}
+
+@media screen and (max-width: 600px){
+	html {
+		font-size: 8px;
```

Comment:
I think you generally make font larger when switching to mobile, this makes the body copy quite hard to read